server/metrics/metrics.go

```diff
@@ -0,0 +1,251 @@
+		Name:        "runs_created_count",
+		Help:        "Total number of runs created since the last launch.",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.runsCreatedCount)
+
+	m.runsFinishedCount = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemRuns,
+		Name:        "runs_finished_count",
+		Help:        "Total number of runs finished since the last launch.",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.runsFinishedCount)
+
+	m.errorsCount = prometheus.NewCounter(prometheus.CounterOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemSystem,
+		Name:        "errors_count",
+		Help:        "Total number of errors since the last launch.",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.errorsCount)
+
+	m.playbooksActiveTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemPlaybooks,
+		Name:        "playbooks_active_total",
+		Help:        "Total number of active playbooks.",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.playbooksActiveTotal)
+
+	m.runsActiveTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemRuns,
+		Name:        "runs_active_total",
+		Help:        "Total number of active runs.",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.runsActiveTotal)
+
+	m.remindersOutstandingTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemRuns,
+		Name:        "reminders_outstanding_total",
+		Help:        "Total number of outstanding reminders.",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.remindersOutstandingTotal)
+
+	m.retrosOutstandingTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemRuns,
+		Name:        "retros_outstanding_total",
+		Help:        "Total number of outstanding retrospective reminders.",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.retrosOutstandingTotal)
+
+	m.followersActiveTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemRuns,
+		Name:        "followers_active_total",
+		Help:        "Total number of active followers.",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.followersActiveTotal)
+
+	m.participantsActiveTotal = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace:   MetricsNamespace,
+		Subsystem:   MetricsSubsystemRuns,
+		Name:        "participants_active_total",
+		Help:        "Total number of active participants (i.e. members of the playbook run channel when the run is active).",
+		ConstLabels: additionalLabels,
+	})
+	m.registry.MustRegister(m.participantsActiveTotal)
+	return m
+}
+
+func (m *Metrics) IncrementPlaybookCreatedTotal(num int) {
```

Comment:
We should probably rename all these functions to use Count instead of Total, too
```suggestion
func (m *Metrics) IncrementPlaybookCreatedCount(num int) {
```